pkg/frontend/validate.go

```diff
@@ -202,20 +202,20 @@ func validateAdminMasterVMSize(vmSize string) error {
 
 // validateInstallVersion validates the install version set in the clusterprofile.version
 // TODO convert this into static validation instead of this receiver function in the validation for frontend.
-func (f *frontend) validateInstallVersion(ctx context.Context, doc *api.OpenShiftCluster) error {
+func (f *frontend) validateInstallVersion(ctx context.Context, oc *api.OpenShiftCluster) error {
```

Comment:
is there a need to change doc to oc?

Reply:
This was a comment left from the last PR made, saying that since its no longer an openshiftclusterdoc, we should change it to oc